### ISSUE TITLE
Windows 10 support ?

### ISSUE THREAD

**Issue by LeG3nDz** [dylang/grunt-notify#108]
Hi community,

Do you know if grunt-notify is worked on Windows 10 ? I've not notif with this config :

    module.exports = {
        sass: {
            options: {
                title: '"Sass"',
                message: '"Compilé avec succès"'
            }
        },
        concat: {
            options: {
                title: '"JavaScript"',
                message: '"Compilé avec succès"'
            }
        },
        prod: {
            options: {
                title: '"Production"',
                message: '"Effectuée avec succès"'
            }
        }
    };

Do you have idea ? Thank you !

**Comment by bxyoung89** [comment]
Second this issue. I had a working config on windows 8. Recently updated and grunt-notify stopped working.

**Comment by yahyazini** [comment]
+1

**Comment by stephenbe** [comment]
To add to this issue,

I can see the notifications in the action center : 

![2015-08-25_09-52-57](https://cloud.githubusercontent.com/assets/3018441/9468348/291785b2-4b0f-11e5-83c2-6d2eb5b8cad0.png)

**Comment by bxyoung89** [comment]
Looks like you can toggle if the notifications from the action center show up or not. You just need to toggle the settings.

![grunt notifications](https://cloud.githubusercontent.com/assets/5355035/9477757/418ebad2-4b39-11e5-99b4-113e47a6077e.PNG)

Here's the workflow I used:

1. Click on the action center icon (the word bubble thing).
2. Click on "All Settings"
3. Click on "System"
4. Click on "Notifications & Actions"
5. Click on the settings for "toast"
6. Switch "Show notification banners" to on.

After doing that you should see a banner show up in the bottom right corner.


After discovering this, I think dylang should close the issue as notifications are now configurable in Windows 10.

**Comment by stephenbe** [comment]
Omg this worked...THANK YOU! :)

**Comment by LeG3nDz** [comment]
Hi,

This setting is already enabled on my Windows 10 :(

**Comment by stephenbe** [comment]
>Click on the settings for "toast"

Thats the explorer...not toast

**Comment by LeG3nDz** [comment]
Sorry,  not the good screenshot :) This :

![sans-titre-1](https://cloud.githubusercontent.com/assets/4712791/9487840/a789b5e8-4bd5-11e5-9343-23408bc47c4e.jpg)

**Comment by bxyoung89** [comment]
I'd make sure you're getting notifications in the center at all like stephenbe's picture. If you're not, there's probably something not quite right with your use of grunt-notify. If they are showing up, maybe there's another setting you need to change. You might have all notifications muted or something like that.

**Comment by yahyazini** [comment]
Thanks @bxyoung89 that fixed it!

**Comment by Ouro17** [comment]
Yep, as @bxyoung89 said, that way works, I didnt know that I can click on "toast" element, they should make it more clear, thank you!
